src/backend/services/auth/request_validators.py

```diff
@@ -0,0 +1,39 @@
+from fastapi import HTTPException, Request
+
+from backend.services.auth.jwt import JWTService
+
+
+def validate_authorization(request: Request):
```

Comment:
nit: a few functions don't have the return type

Reply:
done